build-support/src/main/resources/app/cash/redwood/buildsupport/flexboxLayout.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example
+
+import app.cash.redwood.flexcontainer.AlignItems
```

Comment:
I'd probably rename this file to `flexContainer` or something similar to match the package names of the other modules. I migrated all flexbox references to flex container a while back.

Reply:
Yes. I will change.

Although... I've been biding my time for when to bring up that my original proposal was that the package name and project name should still be 'flexbox' and its only the type name that I thought should be `FlexContainer`. But that's a separate thing.

Reply:
Changing the package back to `flexbox` sounds fine to me!